README.md

```diff
@@ -100,6 +100,8 @@ for {
 
 In the next two sections the core classes used by this library are presented. They are followed by a description of the full capabilities of the DSL, including the ability to parse content after extracting, validating the contents of a page and defining custom extractors or validators.
 
+The jsoup cook book provides a good overview on [CSS selector syntax](https://jsoup.org/cookbook/extracting-data/selector-syntax) with a more detailed reference available in the [API docs](https://jsoup.org/apidocs/org/jsoup/select/Selector.html).
```

Comment:
A small paragraph with some reference on CSS selectors and queries sounds great to me, but:

- Could you move this paragraph to before the previous one (the one starting with "In the next two sections...")? That one should be the last since it introduces the next sections;
- I'd prefer not to reference jsoup docs, since jsoup is only one of the backend implementations for scala-scraper. Clients should not rely on jsoup-specific behaviors or APIs, but there are very good general CSS resources too, like https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors. Could you change the links to reflect that?

Moreover, documentation for this project is generated using tut. You should change core/src/main/tut/README.md instead and run `sbt tut` afterwards.